liblangutil/Scanner.cpp

```diff
@@ -100,7 +101,31 @@ int hexValue(char c)
 }
 } // end anonymous namespace
 
+std::string to_string(ScannerError _errorCode)
+{
+	switch (_errorCode) {
+		case ScannerError::NoError: return "No error.";
+		case ScannerError::IllegalToken: return "Cannot lexically analyze the token.";
```

Comment:
I don't think people understand what this means.

Reply:
confirmed (I have difficulties...)